### PR TITLE
Push notification vacation reminder

### PR DESCRIPTION
**Pull request by sponkrashin**
Issue #458

### CODE REVIEW

File: server/Arcadia.Assistant.CSP/Vacations/CspEmployeeVacationsRegistry.cs [hunk 1]
```diff
@@ -47,7 +50,7 @@ protected override void OnReceive(object message)
                     break;
 
                 case Initialization.Success msg:
-                    foreach (var @event in msg.Events.Where(e => e.CalendarEvent.IsPending))
+                    foreach (var @event in msg.Events)
```
Comment (santee) [inline server/Arcadia.Assistant.CSP/Vacations/CspEmployeeVacationsRegistry.cs]: intended?

File: server/Arcadia.Assistant.CSP/Vacations/CspEmployeeVacationsRegistry.cs [hunk 2]
```diff
@@ -22,16 +23,18 @@ public class CspEmployeeVacationsRegistry : UntypedActor, ILogReceive
...
         {
             this.vacationsSyncExecutor = vacationsSyncExecutor;
             this.employeeId = employeeId;
 
-            Context.System.Scheduler.ScheduleTellOnce(
-                TimeSpan.Zero,
-                this.Self,
-                Initialization.Instance,
-                this.Self);
+            // Not better place to create it, but I don't know where we can do it else
```
Comment (santee) [inline server/Arcadia.Assistant.CSP/Vacations/CspEmployeeVacationsRegistry.cs]: ok

File: server/Arcadia.Assistant.CSP/Vacations/CspEmployeeVacationsRegistry.cs (hunk 1 above)
Comment (sponkrashin) [reply server/Arcadia.Assistant.CSP/Vacations/CspEmployeeVacationsRegistry.cs]: Yes. Some actors want to know about only pending events recovery, but reminder wants to know about approved events. That's why I moved this logic to concrete receiver of event bus message.

File: server/Arcadia.Assistant.CSP/Vacations/EmployeeVacationApprovedAccountingReminderActor.cs [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+                    this.Unhandled(message);
+                    break;
+            }
+        }
+
+        private void OnEventReceived(CalendarEvent @event)
+        {
+            if (@event.Status != VacationStatuses.Approved)
+            {
+                if (this.remindedEvents.ContainsKey(@event.EventId))
```
Comment (santee) [inline server/Arcadia.Assistant.CSP/Vacations/EmployeeVacationApprovedAccountingReminderActor.cs]: not remindedEvents in this case but vacationsToRemind?

Comment (sponkrashin) [reply server/Arcadia.Assistant.CSP/Vacations/EmployeeVacationApprovedAccountingReminderActor.cs]: Done.

**Pull request merged**